netmiko/snmp_autodetect.py

```diff
@@ -59,7 +59,7 @@
     },
     "cisco_xe": {
         "oid": ".1.3.6.1.2.1.1.1.0",
-        "expr": re.compile(r".*IOS-XE Software,.*", re.IGNORECASE),
+        "expr": re.compile(r".*Catalyst L3 Switch Software.*", re.IGNORECASE),
```

Comment:
hmmmm, I don't think that will work (i.e. that is too generic and would match platforms that should be categorized as `cisco_ios`)